x-pack/test/functional/page_objects/infra_hosts_view.ts

```diff
@@ -129,8 +129,6 @@ export function InfraHostsViewProvider({ getService }: FtrProviderContext) {
       const button = await element.findByTestSubject('embeddablePanelToggleMenuIcon');
       await button.click();
       await testSubjects.existOrFail('embeddablePanelAction-openInLens');
-      // forces the modal to close
-      await element.click();
```

Comment:
I think somehow this click opened the modal and this test failed multiple times:
<img width="1527" alt="image" src="https://github.com/user-attachments/assets/5781db4c-f05f-4c87-bc15-e9411954c47b">


I don't know why it happens only on this PR - my assumption is that with a different TX formula now it clicks on the chart line 🤔 

In the end, I used an escape button key press to close it in [hosts_view.ts](https://github.com/elastic/kibana/pull/189281/files#diff-cdf2ef63d7f0db051d569b09fbd2ba39eb852d3d75bf3b29d85e57de2da3af35) to avoid unexpected behavior